src/silo/config/preprocessing_config.cpp

```diff
@@ -14,13 +14,21 @@ void PreprocessingConfig::validate() const {
    if (!std::filesystem::exists(input_directory)) {
       throw preprocessing::PreprocessingException(input_directory.string() + " does not exist");
    }
-   if (ndjson_input_filename.has_value() && metadata_file) {
+   if (ndjson_input_filename.has_value() && metadata_file.has_value()) {
       throw preprocessing::PreprocessingException(fmt::format(
          "Cannot specify both a ndjsonInputFilename ('{}') and metadataFilename('{}').",
          ndjson_input_filename.value().string(),
          metadata_file.value().string()
       ));
    }
+   if (!ndjson_input_filename.has_value() && !metadata_file.has_value()) {
+      throw preprocessing::PreprocessingException(fmt::format(
+         "Neither a ndjsonInputFilename ('{}') nor a metadataFilename ('{}') was specified as "
```

Comment:
Here you could list the various ways in which they can be passed, but not super important (CLI, env, config file)

Reply:
I removed it after the above comment.

We have them described in the [README](https://github.com/GenSpectrum/LAPIS-SILO?tab=readme-ov-file#configuration-files). Although the part about default values and the configs therefore being optional is unfortunately no longer valid as of [my refactoring here](https://github.com/GenSpectrum/LAPIS-SILO/commit/847ec7e8af6eda73c95076c9b4428ed3fd5c3f9f#diff-b2ac6b3e28c6241fa5fce2e67f54fd37358cd2a7e7e0f3c5becab39bd4cf8015L39)...